Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/DaysSinceOnsetOfSymptomsVectorDeterminator.kt

```diff
@@ -0,0 +1,40 @@
+package de.rki.coronawarnapp.submission
+
+import de.rki.coronawarnapp.submission.TransmissionRiskVectorDeterminator.Companion.numberOfDays
+
+class DaysSinceOnsetOfSymptomsVectorDeterminator {
+
+    @Suppress("MagicNumber")
+    internal fun determine(symptoms: Symptoms, size: Int): DaysSinceOnsetOfSymptomsVector {
```

Comment:
Proposal: I would consider dropping the `size` parameter. I don't think we have any scenario where this is _not_ 15.